rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/smeup/MULANGT02ConstAndDSpecTest.kt

```diff
@@ -291,6 +294,8 @@ open class MULANGT02ConstAndDSpecTest : MULANGTTest() {
      */
     @Test
     fun executeMUDRNRAPU00206() {
+        smeupConfig = turnOnZAddLegacyFlagConfig
```

Comment:
In my opinion, this assignment is unhelpful. You should pass the instance of `Configuration` returned by `turnOnZAddLegacyFlagConfig` to the `outputOf` function.
Follow this suggestion also for the other methods where you used this pattern.